crypto/src/lib.rs

```diff
@@ -106,21 +98,23 @@ impl FromStr for Algorithm {
 }
 
 /// Options for key generation
+#[cfg(not(feature = "ffi_import"))]
 #[derive(Debug, Clone)]
-pub enum KeyGenOption {
+enum KeyGenOption {
```

Comment:
Why hide it?

Reply:
because it's just a container for values set on  `KeyGenConfiguration`. User can never interact with it